### PR TITLE
[nnc] tensorexpr for quantized/aten::upsample_nearest2d

### PR DESCRIPTION
**Pull request by IvanKobzarev**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #71243
* __->__ #71236

Differential Revision: [D33553305](https://our.internmc.facebook.com/intern/diff/D33553305)

### CODE REVIEW

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33553305).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33553305).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33553305).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33553305).

File: test/cpp/tensorexpr/test_quantization.cpp [hunk 1]
```diff
@@ -227,6 +227,35 @@ TEST_F(Quantization, QuantUpsampleNearst2dDequantUInt8) {
   CHECK_EQ(check, 1);
 }
 
+TEST_F(Quantization, UpsampleNearst2d) {
```
Comment (ZolotukhinM) [inline test/cpp/tensorexpr/test_quantization.cpp:230]: Typo: Near **e** st

**Review by ZolotukhinM — APPROVED** [review]
Looks good! It might be good to also add a test to sweep all possible dtypes/devices, like it's done for some other ops:
https://github.com/pytorch/pytorch/blob/c5df294940cc20b1f276193a481e09df3199354c/test/test_jit_fuser_te.py#L1341-L1342

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33553305).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33553305).

**Pull request closed without merging**